cypress/integration/interface_builder_test.js

```diff
@@ -711,10 +711,10 @@ describe('Interface builder tests', () => {
 
           const mappingEndpoint = '/new_mapping_endpoint';
 
-          // Add new mapping
-          cy.get('button').contains('Add new mapping...').click();
+          // Add Mapping
+          cy.get('button').contains('Add mapping...').click();
           cy.get('.modal.show').within(() => {
-            cy.get('.modal-header').contains('Add new mapping');
+            cy.get('.modal-header').contains('Add Mapping');
             cy.get('#mappingEndpoint').paste(mappingEndpoint);
             cy.get('#mappingType').select('double');
             cy.get('button').contains('Confirm').click();
```

Comment:
We should check this buttons becomes "Add mapping", in line with other modals

Reply:
Thanks, that should be fixed now